### PR TITLE
feat: add first blog post into docs site

### PR DESCRIPTION
**Pull request by connor4312**
Requires https://github.com/mixer/developer-site/pull/4. Moves the one engineering blog post we have published onto the dev site.

### CODE REVIEW

File: pages/03.content/02.blog/_bios/copeet/bio.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Connor Peet
+image: https://avatars1.githubusercontent.com/u/2230985?s=128
+social:
```
Comment (ProbablePrime) [inline pages/03.content/02.blog/_bios/copeet/bio.md]: We can probably store these variables in the user configuration.

Comment (connor4312) [reply pages/03.content/02.blog/_bios/copeet/bio.md]: Eh. Having them as content-only lets people who don't have grav admin access author blog posts. Maybe we have guest days with posts written by 3rd party developers for instance :)

Comment (ProbablePrime) [reply pages/03.content/02.blog/_bios/copeet/bio.md]: We can give guests admin access, there's a full permission system.

Comment (ProbablePrime) [reply pages/03.content/02.blog/_bios/copeet/bio.md]: Sure

**Pull request merged**